### ISSUE TITLE
"Application Error" Error message

### ISSUE THREAD

**Issue by TSalazargr** [HabitRPG/habitica#6561]
I'm browsing habitica on chrome and I keep getting the following message: "Application Error
An error occurred in the application and your page could not be served. Please try again in a few moments.

If you are the application owner, check your logs for details."

**Comment by untonuggan** [comment]
I am having the same issue, and it's repeated several times today. I'm running Mozilla Firefox on Fedora Linux.

**Comment by wfischer42** [comment]
I've had that issue several times recently in Chrome. Here's my console screenshot:
![image](https://cloud.githubusercontent.com/assets/16843426/12522477/2f7c4e0e-c10e-11e5-93a9-ed089fd68db4.png)

UID: 9aeea38c-ccff-45ec-a22a-396a8bb35cd4

**Comment by Alys** [comment]
We restarted the server a few hours ago which should have prevented these errors for at least a while. I'm closing this but please comment here if they keep on happening for you and a single reload doesn't fix them.

**Comment by hartopoangeline** [comment]
got this too. Im on safari, macbook pro

![screen shot 2016-01-28 at 9 55 56 pm](https://cloud.githubusercontent.com/assets/16939963/12648078/cff685b0-c60a-11e5-998f-7dd5afb65b8c.png)

Yikes, an error occurred! Please reload the page, your last action may not have been saved correctly.

"Error

s, please report it at Help > Report a Bug. If you're familiar with your browser's console, please include any error messages.

**Comment by Alys** [comment]
@hartopoangeline I've restarted the server which should stop the errors.

**Comment by hartopoangeline** [comment]
thank you :D:D

On Thu, Jan 28, 2016 at 11:54 PM, Alys notifications@github.com wrote:

> [image: Boxbe] https://www.boxbe.com/overview This message is eligible
> for Automatic Cleanup! (notifications@github.com) Add cleanup rule
> https://www.boxbe.com/popup?url=https%3A%2F%2Fwww.boxbe.com%2Fcleanup%3Ftoken%3DrQBxtLJa%252FnLIJ57cUK3UBwzqHNH9Yi8vPdAHvVCRH0Rx8zC5XQ1aQy4ectjAJnxWppwjbOeh61zrMH6%252BH852tsJj5%252B7AkZL4K9Y%252Fd05beVqoa5%252FOyMFZCeGbf7mmh8nhYDLZrwkAlUbAhre4PdY2qA%253D%253D%26key%3DF9NsLjNVkaU8gWESlR4CXgiQ0ql66S9svxz70BU%252FfeQ%253D&tc_serial=24175022026&tc_rand=2082746216&utm_source=stf&utm_medium=email&utm_campaign=ANNO_CLEANUP_ADD&utm_content=001
> | More info
> http://blog.boxbe.com/general/boxbe-automatic-cleanup?tc_serial=24175022026&tc_rand=2082746216&utm_source=stf&utm_medium=email&utm_campaign=ANNO_CLEANUP_ADD&utm_content=001
> 
> @hartopoangeline https://github.com/hartopoangeline I've restarted the
> server which should stop the errors.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/HabitRPG/habitrpg/issues/6561#issuecomment-176280703.

## 

_Regards,_

Angeline H.
Universitas Pelita Harapan
Visual Communication Design Student, 2012

**Comment by einahpets65** [comment]
i'm still having this issue when I am on chrome

**Comment by Alys** [comment]
@einahpets65 How often are the errors appearing?

**Comment by einahpets65** [comment]
@Alys every time I try to use any page, and has been for the past few days. It takes a few minutes for the loading bar to fill, and then the error that hartopoangeline posted a picture of pops up

**Comment by Alys** [comment]
@einahpets65 **Click the button on this page: https://habitica.com/static/clear-browser-data
If that doesn't help, try a different browser or incognito / private mode in the same browser. Tell us how that goes.**

**Comment by einahpets65** [comment]
@alys it worked! Awesome, thank you so much!

**Comment by einahpets65** [comment]
@alys Hmm, well it worked for a second, but now it's not even letting me log in anymore. I can access the site through firefox, but obviously I would ideally like to be able to access it through chrome because that's the browser I actually use.

**Comment by emalven** [comment]
I am having the same error using Safari on a Macbook Pro.
[Habitica Error.pdf](https://github.com/HabitRPG/habitrpg/files/114086/Habitica.Error.pdf)

**Comment by Alys** [comment]
@einahpets65 _"it's not even letting me log in"_ -- What's happening when you try?

@emalven Have you tried the suggestions in my post five up from here? I've just bolded the text to make it easier to find.

**Comment by einahpets65** [comment]
Occasionally the log in form will pop up, but nothing happens when I click on the log in button. Usually though, I click on the blue Play Habatica button, and the blue loading bar appears, but the log in form never does so I'm not able get to the point where I can put in my username or password.

**Comment by einahpets65** [comment]
update: I went to my chrome settings and made sure when i cleared my browser history from the beginning of time and did the habatica clear browser data button link as well and it appears to be working again

**Comment by Jathay** [comment]
For the last hour or so I haven't been able to do anything on the website due to application errors. They appear every time I load/reload the page, even in incognito mode and after I cleared the habitica browser data like Alys recommended above. I had to use the app to add a to-do, and I can't do anything on the website between the application error and an error 503 that popped up after trying to sign in (after clearing the browser data). I've got some screen grabs of what it looks like.
![habiticaapplicationerror1](https://cloud.githubusercontent.com/assets/17188442/12996705/3faf0dfa-d100-11e5-859c-ba31a510cd4a.jpg)
![habiticaapplicationerror2](https://cloud.githubusercontent.com/assets/17188442/12996709/434925d6-d100-11e5-94d7-a8522c74e9bd.jpg)
